x-pack/test/security_solution_cypress/cypress/e2e/cloud_security_posture/misconfiguration_contextual_flyout.cy.ts

```diff
@@ -0,0 +1,216 @@
+  'securitySolutionFlyoutMisconfigurationFindingsTable'
+);
+
+const timestamp = Date.now();
+
+// Create a Date object using the timestamp
+const date = new Date(timestamp);
+
+// Convert the Date object to ISO 8601 format
+const iso8601String = date.toISOString();
+
+const mockFindingHostName = (matches: boolean) => {
+  return {
+    '@timestamp': iso8601String,
+    host: { name: matches ? 'siem-kibana' : 'not-siem-kibana' },
+    resource: {
+      id: '1234ABCD',
+      name: 'kubelet',
+      sub_type: 'lower case sub type',
+    },
+    result: { evaluation: matches ? 'passed' : 'failed' }, // Adjusting result based on matches
+    rule: {
+      name: 'Upper case rule name',
+      section: 'Upper case section',
+      benchmark: {
+        id: 'cis_k8s',
+        posture_type: 'kspm',
+        name: 'CIS Kubernetes V1.23',
+        version: 'v1.0.0',
+      },
+      type: 'process',
+    },
+    cluster_id: 'Upper case cluster id',
+    data_stream: {
+      dataset: 'cloud_security_posture.findings',
+    },
+  };
+};
+
+const mockFindingUserName = (matches: boolean) => {
+  return {
+    '@timestamp': iso8601String,
+    user: { name: matches ? 'test' : 'not-test' },
+    resource: {
+      id: '1234ABCD',
+      name: 'kubelet',
+      sub_type: 'lower case sub type',
+    },
+    result: { evaluation: matches ? 'passed' : 'failed' }, // Adjusting result based on matches
+    rule: {
+      name: 'Upper case rule name',
+      section: 'Upper case section',
+      benchmark: {
+        id: 'cis_k8s',
+        posture_type: 'kspm',
+        name: 'CIS Kubernetes V1.23',
+        version: 'v1.0.0',
+      },
+      type: 'process',
+    },
+    cluster_id: 'Upper case cluster id',
+    data_stream: {
+      dataset: 'cloud_security_posture.findings',
+    },
+  };
+};
+
+const createMockFinding = (isNameMatches: boolean, findingType: 'host.name' | 'user.name') => {
+  return rootRequest({
+    method: 'POST',
+    url: `${Cypress.env(
+      'ELASTICSEARCH_URL'
+    )}/${CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN}/_doc`,
+    body:
+      findingType === 'host.name'
+        ? mockFindingHostName(isNameMatches)
+        : mockFindingUserName(isNameMatches),
+  });
+};
+
+const deleteDataStream = () => {
+  return rootRequest({
+    method: 'DELETE',
+    url: `${Cypress.env(
+      'ELASTICSEARCH_URL'
+    )}/_data_stream/${CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN}`,
+  });
+};
+
+describe('Alert Host details expandable flyout', { tags: ['@ess', '@serverless'] }, () => {
+  beforeEach(() => {
+    deleteAlertsAndRules();
+    login();
+    createRule(getNewRule());
+    visit(ALERTS_URL);
+    waitForAlertsToPopulate();
+  });
+
+  context('Host name - Has misconfiguration findings', () => {
+    beforeEach(() => {
+      createMockFinding(true, 'host.name');
+      cy.reload();
+      expandFirstAlertHostFlyout();
+    });
+
+    afterEach(() => {
+      deleteDataStream();
```

Comment:
here it is worth commenting that we deleting the data stream while we don't create it because the data stream is being created automatically when any `cloud_security_api` is hit